src/core/textures/Texture.js

```diff
@@ -140,6 +140,7 @@ export default class Texture extends EventEmitter
         {
             baseTexture.once('loaded', this.onBaseTextureLoaded, this);
         }
+        baseTexture.on('destroy', this.destroy, this);
```

Comment:
Good morning, memory leak! 

Now people will be forced to destroy temp Texture objects, not only temp BaseTextures.